src/controls/editor-mode/right-sidebar.js

```diff
@@ -155,7 +156,10 @@ const EditorRightSidebar = () => {
                 'Labels',
                 showLabels,
                 v => userUpdate({ genericOptions: { showLabels: v } }),
-                JSON.stringify(renderableValueKeys)?.includes(MAP_VALUE_VIS.elevation)
+                JSON.stringify(renderableValueKeys)?.includes(MAP_VALUE_VIS.elevation) ||
+                (useMVTOption && !(MAP_LAYER_GEO_KEYS.scatterplot.includes(domain.value) ||
+                  GEO_KEY_TYPES.region.includes(domain.value) ||
+                  JSON.stringify(renderableValueKeys)?.includes(MAP_VALUE_VIS.elevation))),
```

Comment:
This is a bit hard to read. Could you consider extracting the computation of this value before the function call?

Also, can we directly look up the elevation value within the Object `renderableValueKeys` instead of doing a JSON stringify and then string lookup? For context, these two are both quite expensive (computationally).

Reply:
Done